source/partials/release_notes/_release-20-6-0.md.erb

```diff
@@ -0,0 +1,68 @@
+<h4>New Pipeline as code wizard</h4>
+
+The Pipeline as code wizard makes it easier for users to define and generate pipeline definitions using the config repo plugin of their choice.
+
+<h4>SSL support - Breaking Changes</h4>
+
+In GoCD 20.2.0 we had [announced](https://github.com/gocd/gocd/issues/7872) the removal of support for TLS on port 8154 using a self-signed certificate. To minimize disruptions, GoCD temporarily provided an option to enable the TLS port.
+
+With this release, GOCD no longer supports configuring TLS within GoCD and will stop bringing up TLS on port 8154. If you have temporarily enabled TLS, and if the agents are configured to use the TLS port 8154 they will have to reconfigured to use port 8153, else the agent will stop connecting to the server.
+
+<h4>Business Continuity</h4>
+
+GoCD had a failover mechanism to decrease the impact of server failure. This feature was available as part of the paid offering through the Business Continuity Addon.
+As part of open sourcing all the commercial components, the Business continuity feature has been incorporated into GoCD core. Please refer to the [documentation](https://docs.gocd.org/20.6.0/advanced_usage/business_continuity/introduction.html) to know more about the Business Continuity feature.
+
+<h4>Data Sharing</h4>
+
+The Data Sharing feature has been removed from GoCD.
+
+<h4>Other Improvements</h4>
+
+* <%= link_to_issue 8210, 'Allow `git` material to honor a refSpec as the branch name' %>
+* <%= link_to_issue 6722, 'Showcase a help component for concepts on all SPAs' %>
+* <%= link_to_issue 8256, 'Improvements on Pipeline Config SPA' %>
```

Comment:
@kritika-singh3 - some of the commits regarding to the pipeline config spa are bug fix. Should we move this section to bug-fix and possibly mention following changes too?

```
cbfe801875 Pipeline config spa fixes (#8141)


546392b856 Fixes: - Add a positive integer validation for timeout and run instance count if integer - showcasing the errors on add job and add/update task modal popup


828f368838 More fixes: - do not send elastic profile id in post if empty - should not validate name or path in custom tab if both of them are empty. otherwise, do validate the presense - added validation for name and stages for template config


4f0ad0fc93 Fixes on new pipeline config spa: - The global error on client validation failure will now go away after 5 secs - Added min="0" for run instance count and job timeout - Added validation for custom tab path when name is specified
```

Reply:
Instead of mentioning multiple commit regarding pipeline config SPA, I would prefer mentioning a single [issue](https://github.com/gocd/gocd/issues/8141) - since all the points mentioned are either fixed (or won't fix). WDYT?